spec/index.html

```diff
@@ -306,4 +306,352 @@ <h3 class="orange">3<sup>rd</sup> Edition DRAFT</h3>
+  <li><a href="http://www.ecma-international.org/ecma-262/6.0/index.html#sec-returnifabrupt">ReturnIfAbrupt</a>(<var>collator</var>).</li>
+  <li>Return CompareStrings(<var>collator</var>, <var>S</var>, <var>That</var>)</li>
+</ol></emu-alg>
+
+      <p>
+        The value of the <emu-val>length</emu-val> property of the <emu-val>localeCompare</emu-val> method is 1.
+      </p>
+
+      <emu-note><span class="note">Note 1</span>
+        <p>
+          The <emu-val>localeCompare</emu-val> method itself is not directly suitable as an argument to <emu-val>Array.prototype.sort</emu-val> because the latter requires a function of two arguments.
+        </p>
+      </emu-note>
+
+      <emu-note><span class="note">Note 2</span>
+        <p>
+          The <emu-val>localeCompare</emu-val> function is intentionally generic; it does not require that its <emu-val>this</emu-val> value be a String object. Therefore, it can be transferred to other kinds of objects for use as a method.
+        </p>
+      </emu-note>
+
+    </emu-clause>
+
+    <emu-clause id="sec-String.prototype.toLocaleLowerCase">
+      <h1><span class="secnum">5.1.2</span>String.prototype.toLocaleLowerCase ([locales])</h1>
+
+      <p>
+        This definition supersedes the definition provided in ES2015, 21.1.3.20.
+      </p>
+
+      <p>
+        This function interprets a string value as a sequence of code points, as described in ES2015, 6.1.4. The following steps are taken:
+      </p>
+
+      <emu-alg aoid="alg-String.prototype.toLocaleLowerCase" id="ao-alg-String.prototype.toLocaleLowerCase"><ol>
+  <li>Let <var>O</var> be <a href="http://www.ecma-international.org/ecma-262/6.0/index.html#sec-requireobjectcoercible">RequireObjectCoercible</a>(<emu-val>this</emu-val> value).</li>
+  <li>Let <var>S</var> be <a href="http://www.ecma-international.org/ecma-262/6.0/index.html#sec-tostring">ToString</a>(<var>O</var>).</li>
+  <li><a href="http://www.ecma-international.org/ecma-262/6.0/index.html#sec-returnifabrupt">ReturnIfAbrupt</a>(<var>S</var>).</li>
+  <li>Let <var>requestedLocales</var> be CanonicalizeLocaleList(<var>locales</var>).</li>
+  <li><a href="http://www.ecma-international.org/ecma-262/6.0/index.html#sec-returnifabrupt">ReturnIfAbrupt</a>(<var>requestedLocales</var>).</li>
+  <li>Let <var>len</var> be the number of elements in <var>requestedLocales</var>.</li>
+  <li>If <var>len</var> &gt; 0, then
+    <ol>
+      <li>Let <var>requestedLocale</var> be the first element of <var>requestedLocales</var>.</li>
+    </ol>
+  </li>
+  <li>Else
+    <ol>
+      <li>Let <var>requestedLocale</var> be DefaultLocale().</li>
+    </ol>
+  </li>
+  <li>Let <var>noExtensionsLocale</var> be the String value that is <var>requestedLocale</var> with all Unicode locale extension sequences (6.2.1) removed.</li>
```

Comment:
cross link to 6.2.1 requires section 6 to be merged, otherwise ecmarkup fails to build.